exchanges/request/request.go

```diff
@@ -337,6 +339,14 @@ func (r *Requester) DoRequest(req *http.Request, path string, body io.Reader, re
 			return err
 		}
 
+		if httpDebug {
+			dump, err := httputil.DumpResponse(resp, true)
+			if err != nil {
+				log.Errorf("DumpResponse invalid response: %v:", err)
+			}
+			log.Debugf("DumpResponse: %q", dump)
+		}
+
 		resp.Body.Close()
 		if verbose {
 			log.Debugf("HTTP status: %s, Code: %v", resp.Status, resp.StatusCode)
```

Comment:
Could we remove this line now that we have such a util?
https://github.com/thrasher-/gocryptotrader/pull/298/files#diff-d0bee293ec050b1dd3e914acdef7f6a6R353
(I cant add a comment to non affected code)

Reply:
Great suggestion!

After discussion DumpResponse has been changed to not include the body to stop the double up